### PR TITLE
[Various] More link fixes

### PR DESCRIPTION
**Pull request by kodster28**

### CODE REVIEW

File: content/cache/about/default-cache-behavior.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ The output of the `CF-Cache-Status header` shows whether or not a resource is ca
         The origin server instructed Cloudflare to bypass cache via a Cache-Control header set to{' '}
         <code>no-cache</code>, <code>private</code>, or <code>max-age=0</code> even though
         Cloudflare originally preferred to cache the asset. BYPASS is returned when enabling{' '}
-        <a href="/about/cache-control">Origin Cache-Control</a>. Cloudflare also sets BYPASS when
+        <a href="/cache//about/cache-control/">Origin Cache-Control</a>. Cloudflare also sets BYPASS when
```
Comment (lukeed) [inline content/cache/about/default-cache-behavior.md]:
```suggestion
        <a href="/cache/about/cache-control/">Origin Cache-Control</a>. Cloudflare also sets BYPASS when
```

File: content/cloudflare-one/analytics/logs/audit-logs.md [hunk 2]
```diff
@@ -154,4 +154,4 @@ Cloudflare Logpush pushes Enterprise customers' HTTP request logs, including Acc
 
 For instructions on setting up Logpush, see [Manage the Logpush API](/logs/logpush/logpush-configuration-api/).
 
-For more on exporting per-request Access logs, see [Understanding the Logpush API](/logs/logpush/logpush-configuration-api/understanding-logpush-api/).
+For more on exporting per-request Access logs, see [Understanding the Logpush API](/logs/reference/logpush-api-configuration/).
```
Comment (lukeed) [inline content/cloudflare-one/analytics/logs/audit-logs.md:157]: assume this is a redirect update?

File: content/distributed-web/ethereum-gateway/about-eth.md [hunk 3]
```diff
@@ -59,13 +59,13 @@ sent to the wider network and added to the consensus.
...
 Cloudflare's Gateway. To learn more about how to do this see [Interacting with
-the Ethereum network](/distributed-web/interacting-with-the-eth-gateway/).
+the Ethereum network](/distributed-web/ethereum-gateway/interacting-with-the-eth-gateway/).
 
 ## Connect your website to the gateway
 
 If you want to be able to access the Ethereum network accessible from a custom
 domain name, you can do that using Cloudflare’s Ethereum Gateway. To
-learn how, refer to [Connecting your Website](/distributed-web/connecting-your-website/).
+learn how, refer to [Connecting your Website](/distributed-web//ethereum-gateway/connecting-your-website/).
```
Comment (lukeed) [inline content/distributed-web/ethereum-gateway/about-eth.md]:
```suggestion
learn how, refer to [Connecting your Website](/distributed-web/ethereum-gateway/connecting-your-website/).
```

**Review by lukeed — APPROVED** [review]
Thanks so much!

File: content/cloudflare-one/analytics/logs/audit-logs.md (hunk 2 above)
Comment (kodster28) [reply content/cloudflare-one/analytics/logs/audit-logs.md:157]: Yep!

**Comment by kodster28** [comment]
Absolutely! There are still a few broken links floating out there, but we're at pre-migration levels now (~30).

**Comment by lukeed** [comment]
Ty sir – working on the CI check now so it shouldn't grow anymore going forward

**Pull request merged**